runtime/onert/backend/gpu_cl/open_cl/ClMemory.cc

```diff
@@ -34,6 +34,8 @@ cl_mem_flags ToClMemFlags(AccessType access_type)
       return CL_MEM_WRITE_ONLY;
     case AccessType::READ_WRITE:
       return CL_MEM_READ_WRITE;
+    default:
+      return CL_MEM_READ_ONLY; // unreachable
```

Comment:
(optional) maybe throw here too;
I'm not sure this is realistic but if someone adds another type in `AccessType` then this may cause a problem if not added.